src/libstd/Cargo.toml

```diff
@@ -24,6 +24,7 @@ compiler_builtins = { version = "0.1.16" }
 profiler_builtins = { path = "../libprofiler_builtins", optional = true }
 unwind = { path = "../libunwind" }
 hashbrown = { version = "0.5.0", features = ['rustc-dep-of-std'] }
+wasi = { version = "0.7.0", features = ['rustc-dep-of-std', 'alloc'] }
```

Comment:
I think this probably wants to be a target-specific dependency that's only compiled for the `wasi` target:

```rust
[target.wasm32-wasi.dependencies]
wasi = { version = "0.7.0", features = ['rustc-dep-of-std', 'alloc'] }
```

Reply:
Ah, forgot about that. Fixed!